utility_analysis_new/combiners.py

```diff
@@ -186,6 +166,26 @@ def explain_computation(self):
         pass
 
 
+@dataclass
+class CountUtilityAnalysisMetrics:
```

Comment:
It's moved here in order to keep structure

1.PartitionSelection
2.Count
3.Sum